src/status_im/contact/core.cljs

```diff
@@ -48,6 +67,16 @@
               (add-new-contact contact)
               (send-contact-request contact))))
 
+(fx/defn add-contact-tag [{:keys [db] :as cofx} whisper-id tag]
```

Comment:
Are these fns used or it's something you will be adding at a later stage?

Reply:
it is something that will be used at a later stage, it's fx producing functions to add/remove a tag on a contact